src/main/java/nom/tam/fits/Header.java

```diff
@@ -908,6 +909,43 @@ public HeaderCard findCard(String key) {
         return card;
     }
 
+    /************************************
+    * brief Collect the header cards that match a regular expression.
+    *
+    * This is useful if one needs to search for a keyword that is buried under
+    * some HIERARCH string conventions of unspecified depth. So to search for
+    * some key like "HIERARCH OBO SUBOBO MYOBO", which would appear with the
+    * key HIERARCH.OBO.SUBOBO.MYOBO in this FITS implementation, one could
+    * search with regex="HIER.*MYOBO" and find it, supposed FitsFactory.setUseHierarch(true)
+    * was called before creating the header.
+    * 
+    * @param regex The generalized regular expression for the keyword search
+    * @return The list of header cards that match the regular expression.
+    * @author Richard J. Mathar
+    * @since 2015-03-03
```

Comment:
`@since` is not correct for the library. I recommend you change it to `@since 1.19.1`.